lib/cheerio.js

```diff
@@ -29,11 +29,11 @@ var api = [
  * @param {string|cheerio|node|node[]} [root] - Sets the root node.
  * @param {object} [options] - Options for the instance.
  *
- * @mixes attributes
- * @mixes css
- * @mixes forms
- * @mixes manipulation
- * @mixes traversing
+ * @mixes module:cheerio/attributes
+ * @mixes module:cheerio/css
+ * @mixes module:cheerio/forms
+ * @mixes module:cheerio/manipulation
+ * @mixes module:cheerio/traversing
```

Comment:
This is the other big change: All of these are now based on modules, removing some previously needed hacks.